simpleclient_spring_web/src/main/java/io/prometheus/client/spring/web/MethodTimer.java

```diff
@@ -25,8 +28,8 @@
 @Scope("prototype")
 @ControllerAdvice
 public class MethodTimer {
-    private final ReadWriteLock summaryLock = new ReentrantReadWriteLock();
-    private final HashMap<String, Summary> summaries = new HashMap<String, Summary>();
+    private final ReadWriteLock collectorLock = new ReentrantReadWriteLock();
+    private final HashMap<String, SimpleCollector> collectors = new HashMap<String, SimpleCollector>();
```

Comment:
SimpleCollector is intended as an implementation detail, it shouldn't be used directly.

Reply:
Can definitely change this to `Collector` if we move forward with this PR. Thanks!